### PR TITLE
Drop `enable-tracing` property from Spring Boot integration docs.

### LINKED ISSUE #3289: Java spring boot `enable-tracing` is gonna be deprecated
https://docs.sentry.io/platforms/java/guides/spring-boot/performance/#enable-tracing

related to https://github.com/getsentry/sentry-java/pull/1337

cc @maciejwalkowiak

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
Fixes #3289

### CODE REVIEW

**Review by PeloWriter — COMMENTED** [review]
LGTM

**Comment by marandaneto** [comment]
we need to wait for the Java SDK 4.4 GA, making it a draft to not merge by accident

**Review by marandaneto — APPROVED** [review]

**Comment by marandaneto** [comment]
@maciejwalkowiak I guess its worth an entry to https://docs.sentry.io/platforms/java/migration/

**Comment by maciejwalkowiak** [comment]
@marandaneto migration entry added.

File: src/platforms/java/migration.mdx [hunk 1]
```diff
@@ -4,6 +4,12 @@ sidebar_order: 1000
 description: "Learn about migrating from io.sentry:sentry 1.x to io.sentry:sentry 3.x."
 ---
 
+## Migrating from `io.sentry:sentry` `4.3.0` to `io.sentry:sentry` `5.0.0`
+
+### Spring Boot
+
+- the property `sentry.enable-tracing` is deprecated. To enable tracing simply set `sentry.traces-sample-rate` or create a bean implementing `TracesSamplerCallback`
```
Comment (marandaneto) [inline src/platforms/java/migration.mdx:11]: this could also be a different PR, unless `Enable Tracing` is already deprecated on 4.3.0 so it'd be fine to merge as it is

Comment (maciejwalkowiak) [reply src/platforms/java/migration.mdx:11]: I am not sure what do you mean

Comment (marandaneto) [reply src/platforms/java/migration.mdx:11]: what I mean is, the migration page is necessary to be live in the docs, so people can try the betas

but the removal from platforms/common/performance/index.mdx  cannot be merged yet, because its not stable yet, so they should be different PRs.

Comment (maciejwalkowiak) [reply src/platforms/java/migration.mdx]: Ok, done: https://github.com/getsentry/sentry-docs/pull/3522

**Comment by marandaneto** [comment]
@maciejwalkowiak can you check the conflicts, please?
